nextflow.config

```diff
@@ -71,7 +71,7 @@ params {
     multiqc_methods_description = null
 
     // Boilerplate options
-    outdir                     = null
+    outdir                     = './results'
```

Comment:
outdir is defined twice.

Reply:
What does 'boiler plate options' mean? Are these the options that are included in the template when setting up a novel pipeline? 
Should this line 74 then be removed completely instead of changing it to`outdir = null`, as it would be redundant with line 14?

Reply:
Yes, to both. (I thought we would lint against duplicated entries but maybe not yet)